server/manifest.js

```diff
@@ -36,6 +36,20 @@ module.exports = {
         },
         {
             plugin: {
+                register: 'bassmaster',
+                options: {
+                    batchEndpoint: '/',
+                    tags: ['bassmaster']
+                }
+            }
+        },
+        {
+            plugin: {
+                register: './boilerplate_swagger'
```

Comment:
haute-couture (arbitrarily) prefers `hyphen-case` for directory names.  Might make sense to keep consistency here with whatever changes happen with #9.